libs/remote_storage/src/lib.rs

```diff
@@ -284,7 +284,10 @@ impl GenericRemoteStorage {
     pub fn from_config(storage_config: &RemoteStorageConfig) -> anyhow::Result<Self> {
         Ok(match &storage_config.storage {
             RemoteStorageKind::LocalFs(root) => {
-                info!("Using fs root '{}' as a remote storage", root.display());
+                info!(
+                    "Using fs root '{}' as a remote storage",
+                    root.as_std_path().display()
+                );
```

Comment:
```suggestion
                info!(
                    "Using fs root '{root}' as a remote storage",
                );
```

Unless I am mistaken.